Submodels/MTSOL_call.py

```
@@ -1,20 +1,621 @@
+                break
+            
+            # Once the solution is written to the state file, return the completed exit flag
+            elif 'Solution written to state file' in line and type == 2:
+                exit_flag = ExitFlag.COMPLETED.value
+                break
+            
+            # Once the forces file is written, return the completed exit flag. 
+            # This can be detected from the prompt to overwrite the file or to enter a filename
+            elif line.startswith((' File exists.  Overwrite?  Y', 'Enter filename')) and type == 2:
+                exit_flag = ExitFlag.COMPLETED.value
+                break
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)